tools/pipelines/templates/include-generate-notice-steps.yml

```diff
@@ -8,6 +8,14 @@ parameters:
   type: boolean
 
 steps:
+- task: ComponentGovernanceComponentDetection@0
```

Comment:
```suggestion
# Note: even though we get component governance tasks auto-injected through the 1ES pipeline template, this one is
# currently necessary because the NOTICE file generation task right below requires that CG has run first.
# We're trying to see if we could leverage the auto-injected ones for that.
- task: ComponentGovernanceComponentDetection@0
```